.travis.yml

```diff
@@ -1,4 +1,11 @@
 language: objective-c
-osx_image: xcode9.2
-script:
-  - xcodebuild -project Meniga/Meniga-ios-sdk.xcodeproj -scheme Meniga-ios-sdk-tests -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 8' test
+osx_image: xcode10.2
+
+jobs:
+  include:
+    - stage: test
+      name: "Unit tests"
+      script:
+        - xcodebuild -project Meniga/Meniga-ios-sdk.xcodeproj -scheme Meniga-ios-sdk-tests -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 8' test
```

Comment:
You could put this command in multiple lines so it's easier to see changes later on

Reply:
Changed